src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/auth/MontaguAPIClientTests.kt

```diff
@@ -0,0 +1,45 @@
+package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.auth
```

Comment:
These tests aren't new, they were previously in `MontaguAuthenticationTests` but I moved them into their own class!